src/client/src/ui/analytics/components/analyticsBarChart/PNLBar.tsx

```diff
@@ -0,0 +1,109 @@
+import numeral from 'numeral'
+import React from 'react'
+import { styled } from 'rt-theme'
+export interface PNLBarProps {
+  basePnl: number
+  maxVal: number
+  symbol: string
+}
+
+export const TRANSLATION_WIDTH: number = 50
+
+export const getLogRatio: (max: number, numb: number) => number = (max, numb) => {
+  const logMax = Math.log10(Math.abs(max)) + 1
+  const logNumb = Math.log10(Math.abs(numb))
+  return logNumb / logMax
+}
+export default class PNLBar extends React.Component<PNLBarProps> {
+  render() {
+    const { symbol, basePnl, maxVal } = this.props
+    console.log(symbol, basePnl, maxVal)
+    const color = basePnl >= 0 ? 'green' : 'red'
+    const distance = getLogRatio(maxVal, basePnl) * TRANSLATION_WIDTH * (basePnl >= 0 ? 1 : -1)
+    const price = numeral(Math.abs(basePnl)).format('0a')
+    return (
+      <BarChart>
+        <Label>{symbol}</Label>
+        <Offset />
+        <BarPriceContainer>
+          <PriceContainer distance={distance}>
+            <PriceLabel color={color}>{price}</PriceLabel>
+            <DiamondShape color={color} />
+          </PriceContainer>
+          <Bar />
+          <OriginTickWrapper>
+            <OriginTick />
+            <Origin>0</Origin>
+          </OriginTickWrapper>
+        </BarPriceContainer>
+      </BarChart>
+    )
+  }
+}
+
+const BarChart = styled.div`
```

Comment:
I would consider adding a  `analyticsBarChart/styled.tsx` and move these styled components there.  Keeps the line count of this file down, it is consistent with existing `styled.tsx` pattern